readme.txt

```diff
@@ -2,7 +2,7 @@
 Tags: bitcoin, blockonomics, woocommerce, ecommerce, payments
 Requires at least: 3.0.1
 Tested up to: 4.7.3
-Stable tag: 1.2
+Stable tag: 1.5.2
```

Comment:
Revert changes to readme.txt... it is automatically generated by a convertor from README.md